src/soma.cpp

```diff
@@ -18,11 +21,20 @@ Soma::Soma(const Property::PointLevel& point_properties)
 
 
 Point Soma::center() const {
+
+    if (points().empty()) {
+        return {NaN, NaN, NaN};
```

Comment:
Do people check for NaN? If I use Python, I would just expect this to return `None`, and by now, in compiled languages, expect `std::optional` (or language equivalent)

Reply:
yeah, I removed them after thinking twice and also realizing that the volume/surface functions already raise exceptions.